ignite/contrib/handlers/time_profilers.py

```diff
@@ -1,790 +1,23 @@
-            Total                                                                     22.04571
-            -----------------------------------------  -----------------------  --------------
-            Processing took total 11.29543s [min/index: 0.00393s/1875, max/index: 0.00784s/0,
-             mean: 0.00602s, std: 0.00034s]
-            Dataflow took total 16.24365s [min/index: 0.00533s/1874, max/index: 0.01129s/937,
-             mean: 0.00866s, std: 0.00113s]
-
-        """
-        # adopted implementation of torch.autograd.profiler.build_table
-        handler_column_width = max([len(item[0]) for item in results]) + 4  # type: ignore[arg-type]
-        event_column_width = max([len(item[1]) for item in results]) + 4  # type: ignore[arg-type]
-
-        DEFAULT_COLUMN_WIDTH = 14
-
-        headers = [
-            "Handler",
-            "Event Name",
-            "Total(s)",
-            "Min(s)/IDX",
-            "Max(s)/IDX",
-            "Mean(s)",
-            "Std(s)",
-        ]
-
-        # Have to use a list because nonlocal is Py3 only...
-        SPACING_SIZE = 2
-        row_format_lst = [""]
-        header_sep_lst = [""]
-        line_length_lst = [-SPACING_SIZE]
-
-        def add_column(padding: int, text_dir: str = ">") -> None:
-            row_format_lst[0] += "{: " + text_dir + str(padding) + "}" + (" " * SPACING_SIZE)
-            header_sep_lst[0] += "-" * padding + (" " * SPACING_SIZE)
-            line_length_lst[0] += padding + SPACING_SIZE
-
-        add_column(handler_column_width, text_dir="<")
-        add_column(event_column_width, text_dir="<")
-        for _ in headers[2:]:
-            add_column(DEFAULT_COLUMN_WIDTH)
-
-        row_format = row_format_lst[0]
-        header_sep = header_sep_lst[0]
-
-        result = []
-
-        def append(s: str) -> None:
-            result.append(s)
-            result.append("\n")
-
-        result.append("\n")
-        append(header_sep)
-        append(row_format.format(*headers))
-        append(header_sep)
-
-        for row in results[:-3]:
-            # format min/idx and max/idx
-            row[3] = "{}/{}".format(*row[3])  # type: ignore[misc]
-            row[4] = "{}/{}".format(*row[4])  # type: ignore[misc]
-
-            append(row_format.format(*row))
-
-        append(header_sep)
-        # print total handlers time row
-        append(row_format.format(*results[-3]))
-        append(header_sep)
-
-        summary_format = "{} took total {}s [min/index: {}, max/index: {}, mean: {}s, std: {}s]"
-        for row in results[-2:]:
-            row[3] = "{}s/{}".format(*row[3])  # type: ignore[misc]
-            row[4] = "{}s/{}".format(*row[4])  # type: ignore[misc]
-            del row[1]
-            append(summary_format.format(*row))
-        print("".join(result))
+""" ``ignite.contrib.handlers.time_profilers.py`` was moved to ``ignite.handlers.time_profilers``.
```

Comment:
@KickItLikeShika please update https://github.com/pytorch/ignite/blob/6ac2be161ff7d6e40ca278c78ec20e92007aae7d/ignite/contrib/handlers/__init__.py otherwise any import from contrib submodule will show the warning.